tests_e2e/tests/scripts/agent_cgroups_process_check-unknown_process_check.py

```diff
@@ -62,8 +62,8 @@ def disable_agent_cgroups_with_unknown_process(pid):
     Note: System may kick the added process out of the cgroups, keeps adding until agent detect that process
     """
 
-    def unknown_process_found(cpu_cgroup):
-        cgroup_procs_path = os.path.join(cpu_cgroup, "cgroup.procs")
+    def unknown_process_found(cgroup_path):
+        cgroup_procs_path = os.path.join(cgroup_path, "cgroup.procs")
```

Comment:
can we get cgroup.procs directly from systemd api in agent, instead of building path here?